### DISCUSSION TITLE
Ignore certain registryUrls in the config.js

### DISCUSSION THREAD

**Discussion by snowleopard24** [renovatebot/renovate#24411] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Ability to ignore certain registryUrls in the config.js.
Per the documentation -
```
Usually Renovate is able to either (a) use the default registries for a datasource, or (b) automatically detect during the manager extract phase which custom registries are in use.
```
From what I have gathered through the reading of the logs, it appears that in addition to the listed registryURLs in the config.js it is also using nuget.config and preparing a final list of registryURLs which is a super set of those defined in config.js and nuget.js. Is this true?

Why this doesn't work for us? because in nuget.config we use central package management and hence have rules set for specific feeds  which helps us ignore some feeds while using others.

### What you tried so far.

I don't find any configuration option to ignore certain registryURLs from being used.
The only option would be to delete the registry URLs we don't want to be discovered from our nuget.config but then it would defeat the purpose of central package management and separation of feeds based on rules for specific branches.

Please help!

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
You can override registryUrls using packageRules. This is usually the best way instead of trying to block hosts.

If you think that Renovate should have been able to detect the registryUrls fully automatically based on the files in the repository, then it would be even better if you can describe/demonstrate that in a repo for us to look at

**Comment by snowleopard24** [comment]
Like set package rules to pick packages matching a certain pattern which are available only on a given registryURL and not on others. Hmm. That would do. 
Thanks for the suggestion.
